streaming/spark-streaming/examples/consumer/manifests/01_spark_application.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: spark-consumer
+  namespace: spark-operator
```

Comment:
Dont use `spark-operator` as a namespace to avoid confusion, as this name is only used for Operator deployment. You can use individual team names. e.g., streaming-team-a or steaming-team-b etc. Use our existing spark-team-a creation scripts